ansible_collections/arista/avd/plugins/plugin_utils/cv_client/api/arista/event/v1/__init__.py

```diff
@@ -68,7 +68,7 @@ class EventSeverity(aristaproto.Enum):
 
 class ComponentType(aristaproto.Enum):
     """
-    ComponentType describes the type of entity on which the event occured
+    ComponentType describes the type of entity on which the event occurred
```

Comment:
@ClausHolbechArista will the changes to the api be trashed when we upgrade? Maybe we should ignore everything in there?

Reply:
yes, generated files should be excluded.